INSTALL.md

```diff
@@ -27,22 +27,53 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
-OpenMVG_USE_OPENGV (default OFF): Build with openGV for multi-cameras localization.
-OpenMVG_USE_ALEMBIC (default OFF): Build with Alembic file format support
-OpenMVG_USE_OPENCV (default: OFF): Build with openCV
-OPENMVG_REQUIRE_CERES_WITH_SUITESPARSE (default: ON): By default, openMVG requires Ceres builded with SuiteSparse to ensure best performance, but you can make SuiteSpase optional with this flag.
-
-OpenMVG_BUILD_SHARED (default OFF): Build OpenMVG as shared libs (instead of static libs)
-OpenMVG_BUILD_TESTS (default OFF): Build OpenMVG tests
-OpenMVG_BUILD_DOC (default ON): Build OpenMVG documentation
-OpenMVG_BUILD_EXAMPLES (default: ON): Build OpenMVG samples applications (openMVG softwares are still builded)
-OpenMVG_BUILD_OPENGL_EXAMPLES (default OFF): Build OpenMVG openGL examples
-OpenMVG_BUILD_COVERAGE (default OFF): Enable code coverage generation (gcc only)
+* OpenMVG_USE_BOOST (default ON)
+Use Boost library (enable modules like localization/voctree and other features and optimizations.
+
+* BOOST_NO_CXX11 (default OFF)
+If your Boost binaries are compiled without C++11 support, you need to set this option to avoid compilation errors.
+This is most likely to be the case if you use the system packages to install boost.
+
+* OpenMVG_USE_OPENMP (default ON)
+Use OpenMP parallelization (huge impact on performances)
+
+* OpenMVG_USE_CCTAG (default ON)
+Build with CCTag markers support.
+`-DCCTAG_DIR=/path/to/cctag/install/share/cmake/` (where FindCCTag.cmake can be found)
+
+* OpenMVG_USE_OPENGV (default OFF)
+Build with openGV for multi-cameras localization.
+`-DOPENGV_DIR=/path/to/opengv/install/` (where "include" and "lib" folders can be found)
+We recommend: `git clone https://github.com/poparteu/opengv.git --branch=cmake_fix_install`
+
+* OpenMVG_USE_ALEMBIC (default OFF)
+Build with Alembic file format support.
+`-DAlembic_DIR=/path/to/alembic/install/lib/cmake/Alembic/` (where AlembicConfig.cmake can be found)
+With old Alembic versions (<1.6), you need to set many variables: ALEMBIC_ROOT, ALEMBIC_HDF5_ROOT, ALEMBIC_ILMBASE_ROOT, ALEMBIC_OPENEXR_ROOT.
+
+* OpenMVG_USE_OPENCV (default: OFF): Build with openCV
+`-DOpenCV_DIR=/path/to/opencv/install/share/OpenCV/` (where OpenCVConfig.cmake can be found)
+
+* OPENMVG_REQUIRE_CERES_WITH_SUITESPARSE (default: ON)
+By default, openMVG requires Ceres builded with SuiteSparse to ensure best performances but you can make SuiteSparse optional with this flag.
+
+* OpenMVG_BUILD_SHARED (default OFF)
+Build OpenMVG as shared libs (instead of static libs)
+
+* OpenMVG_BUILD_TESTS (default OFF)
+Build OpenMVG tests
+
+* OpenMVG_BUILD_DOC (default ON)
+Build OpenMVG documentation
+
+* OpenMVG_BUILD_EXAMPLES (default ON)
+Build OpenMVG samples applications (openMVG softwares are still builded)
+
+* OpenMVG_BUILD_OPENGL_EXAMPLES (default OFF)
+Build OpenMVG openGL examples
+
+* OpenMVG_BUILD_COVERAGE (default OFF)
+Enable code coverage generation (gcc only)
```

Comment:
while you are at it, you can correct L:80  'General informations or openMVG SfM pipelines' as it does not render well on markdown if splitted into two lines